basic_helpers.frg

```diff
@@ -12,6 +12,14 @@ pred flows_to[cs: Ctrl, o: Object, f : (CallArgument + CallSite)] {
     }
 }
 
+pred flows_to_ctrl[cs: Ctrl, o: Object, f : CallArgument] {
```

Comment:
maybe at some point we should think about more intuitive naming here. Like maybe this should be `influences`.